spec/models/ability_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Ability do
+
```

Comment:
Any rationale for empty lines here? There is dedicated [check in rubycop](http://stackoverflow.com/questions/824349/modify-the-url-without-reloading-the-page) to eliminate such elements. I was browsing some OS ruby projects (such as RoR) and cannot find similar elements there. 

Can we use [this](https://github.com/bbatsov/ruby-style-guide) and [this](https://github.com/bbatsov/rails-style-guide) as style guide for plankrk?